===================================================================
RCS file: /sources/public/libwww/libwww-config.in,v
retrieving revision 1.6
retrieving revision 1.8
diff -u -r1.6 -r1.8
--- libwww-config.in	1999/06/23 18:05:01	1.6
+++ libwww-config.in	2005/03/10 03:16:05	1.8
@@ -1,7 +1,7 @@
 #!/bin/sh
 # (c) COPYRIGHT MIT 1995.
 # Please first read the full copyright statement in the file COPYRIGH.
-# @(#) $Id: libwww-config.in,v 1.6 1999/06/23 18:05:01 frystyk Exp $
+# @(#) $Id: libwww-config.in,v 1.8 2005/03/10 03:16:05 vbancrof Exp $
 #
 # Thanks to Soren Sandmann for providing first version of this script!
 
@@ -48,7 +48,7 @@
       echo -I@includedir@ -I@includedir@/@PACKAGE@ @DEFS@
       ;;
     --libs)
-      echo -L@libdir@ @LWWWXML@ @LWWWZIP@ @LWWWWAIS@ @LWWWSQL@ -lwwwinit -lwwwapp -lwwwhtml -lwwwtelnet -lwwwnews -lwwwhttp -lwwwmime -lwwwgopher -lwwwftp -lwwwfile -lwwwdir -lwwwcache -lwwwstream -lwwwmux -lwwwtrans -lwwwcore -lwwwutils @LWWWMD5@ @LIBS@
+      echo -L@libdir@ @LWWWZIP@ @LWWWWAIS@ @LWWWSQL@ -lwwwinit -lwwwapp @LWWWXML@ -lwwwhtml -lwwwtelnet -lwwwnews -lwwwhttp -lwwwmime -lwwwgopher -lwwwftp -lwwwfile -lwwwdir -lwwwcache -lwwwstream -lwwwmux -lwwwtrans -lwwwcore -lwwwutils @LWWWMD5@ @LWWWSSL@ @LIBS@ 
       ;;
     *)
       echo "${usage}" 1>&2
